### PR TITLE
Send Variation IDs with conversions

### PR DESCRIPTION
**Pull request by jjl014**
There is a failing test but it looks like it's handled in #42

### CODE REVIEW

File: spec/src/modules/tracker.js [hunk 1]
```diff
@@ -850,6 +850,7 @@ describe('ConstructorIO - Tracker', () => {
     const requiredParameters = {
       name: 'name',
       customer_id: 'customer-id',
+      variation_id: 'variation-id',
```
Comment (sblaurock) [inline spec/src/modules/tracker.js:853]: Is `variation_id` a required param? If not, could we add a separate test for optional params (even if it's just this one) for now?

File: README.md [hunk 2]
```diff
@@ -193,6 +193,7 @@ constructorio.tracker.trackConversion('dogs', {
 | --- | --- |
 | `name` | string |
 | `customer_id` | string |
+| `variation_id` | string |
```
Comment (sblaurock) [inline README.md:196]: Can we update to play nicely with https://github.com/Constructor-io/constructorio-client-javascript/pull/43?

File: spec/src/modules/tracker.js (hunk 1 above)
Comment (jjl014) [reply spec/src/modules/tracker.js]: sounds good. updated. 👍

File: README.md (hunk 2 above)
Comment (jjl014) [reply README.md]: Updated, hopefully what i pushed up works fine when everything gets merged into master

File: README.md [hunk 3]
```diff
@@ -193,6 +193,7 @@ constructorio.tracker.trackConversion('dogs', {
 | --- | --- |
 | `name` | string |
 | `customer_id` | string |
+| `variation_id` | string | ✔ |
```
Comment (jjl014) [inline README.md:196]: Even with the checkmark there, it still displays as if it were only two columns. So it should work fine when everything is merged into master 👍

Comment (sblaurock) [reply README.md:196]: Sweet - let me get that merged and then it should make things easier for you over here.

File: spec/src/modules/tracker.js [hunk 4]
```diff
@@ -891,6 +895,43 @@ describe('ConstructorIO - Tracker', () => {
...
+        expect(requestParams).to.have.property('i');
+        expect(requestParams).to.have.property('s');
+        expect(requestParams).to.have.property('c').to.equal(clientVersion);
+        expect(requestParams).to.have.property('_dt');
+        expect(requestParams).to.have.property('name').to.equal(requiredParameters.name);
+        expect(requestParams).to.have.property('customer_id').to.equal(requiredParameters.customer_id);
+        expect(requestParams).to.have.property('result_id').to.equal(requiredParameters.result_id);
+        expect(requestParams).to.have.property('revenue').to.equal(requiredParameters.revenue.toString());
+        expect(requestParams).to.have.property('section').to.equal(requiredParameters.section);
+        expect(requestParams).to.have.property('variation_id').to.equal(optionalParameters.variation_id);
```
Comment (sblaurock) [inline spec/src/modules/tracker.js:924]: 🔥

**Pull request merged**